### ISSUE TITLE
Fix `busy_detector` warning logs 

### ISSUE THREAD

**Issue by fmassot** [quickwit-oss/quickwit#3406]
I see several warnings when testing Quickwit `edge` image on EKS like this:

```
2023-05-25T06:57:36.818Z  WARN quickwit_cli::busy_detector: Thread wasn't parked for 7069µs, is the runtime too busy?
```

While this can be useful to debug issues, it is quite worrying for a user to see such warnings.

We could for the 0.6:
- change the log level to `debug`
- and/or use a metric so that we can monitor.

**Comment by fmassot** [comment]
I suggest to go for a metric and `debug` log level for the `0.6`.

@trinity-1686a if you think we can do better before next week, let us know

**Comment by fulmicoton** [comment]
Assigning to @fmassot just to avoid leaving that without assignee. You can bounce it to someone else!

**Comment by fmassot** [comment]
Fixed by #3428
